build.gradle

```diff
@@ -1,14 +1,14 @@
 buildscript {
     ext {
-        arrow_version = "0.10.5-SNAPSHOT"
+        arrow_version = "0.10.5"
```

Comment:
Thanks for this contribution @1Jajen1 :tada: Would you mind if we consider `0.11.0-SNAPSHOT`? In that way I could add this library as an Arrow library in the same way of the rest of them: it would use configuration from `arrow` repository. And I could add the publication in OSS and it would be released in the next cycle. Thank you so much!!

Reply:
There are two other libraries that this could be annoying with and that I don't really want to move to a snapshot cycle (because they are outside the arrow ecosystem and should probably stay that way). The prettyprinter and the parser combinators that I use:
- https://github.com/1Jajen1/kotlin-pretty
- https://github.com/1Jajen1/Kparsec

Both use arrow for their api and internals so idk how I'd prevent a breaking change in a snapshot (and 0.11 is going to have a few) to not break this build without updating them first. If there is something that can be done from the config side of things I'd love to change that^^ (I need to properly update and release kparsec anyway sometime soon).

Other than that I like the idea of using the same release cycle :+1:

Reply:
Oh, I understand :+1: We could do the same as it's happening with KotlinTest (KoTest now). It's using an old version of Arrow so we exclude it when adding that dependency. We could try it. I mean, add `0.11.0-SNAPSHOT` to update Arrow Check while Arrow is excluded from prettyprinter and the parser combinators when adding those dependencies. We could try it to see if it's possible. What do you think about it?

Reply:
Worth a try :+1: What needs to be done to move to that lifecycle? I haven't looked at the new gradle configs all that much :see_no_evil:

Reply:
:sweat_smile: don't worry! Great! I'll prepare a PR on this branch to see if it's possible :+1:

Reply:
Thanks! :+1: The build will likely break because of the io changes, so when the config is done I can try and get that fixed^^